vite.config.ts

```diff
@@ -31,23 +30,27 @@ const serverSettings = {
     },
 };
 
-export default defineConfig({
+export default defineConfig((config) => ({
     plugins: [
         react(),
         eslint({
-            failOnWarning: true,
+            failOnWarning: config.mode !== 'development',
+            lintOnStart: config.command === 'serve',
```

Comment:
It's false by default no ?

Reply:
yep

Reply:
Ok I didn't know what it was doing until I read the description of the US.
Should we just set it to `true`, why keeping it `false` when building ?

Reply:
I did that to not modify the behavior with other commands. But I haven't problems with that.

Reply:
> Check all matching files on project startup, too slow, turn on discreetly.

maybe yes